### PR TITLE
fix for brew dependency. add in example android/iPhone

### PR DESCRIPTION
**Pull request by gpx1000**

### CODE REVIEW

File: Formula/airmapd.rb [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+  url "https://github.com/airmap/platform-sdk.git"
   version "0.0.1"
 
   depends_on "cmake" => :build
   depends_on "boost"
   depends_on "c-ares"
   depends_on "pkgconfig"
   depends_on "openssl"
-  depends_on "qt@5.7"
+  depends_on "qt"
```
Comment (vosst) [inline Formula/airmapd.rb]: Hmmm, this seems to be more likely to break if you are not choosing a specific version. In particular as you are passing a specific path further down.

Comment (gpx1000) [reply Formula/airmapd.rb]: Took care of this via symbolic link

**Comment by gpx1000** [comment]
Yes, kinda.  What it does is the XCode version handles loading up, executing with the right environment settings the requirement to have airmapd build an iOS build.  I intentionally am splitting up the PR along these lines (i.e. the fetch dependency call works independently of the downloaded git version, this enables me to merge your PR with my changes that do enable iOS building of airmapd and submit it as a separate PR).
Thus this PR is the client that makes use of the API that will be in the next API :), purely to try to limit the scope of this PR.

File: Formula/airmapd.rb (hunk 1 above)
Comment (gpx1000) [reply Formula/airmapd.rb]: qt brew formula sets the symbolic link when it's installed.  If you look a few lines down, you'll notice I am linking against /usr/local/opt/qt.  That's a symbolic link that the QT brew formula creates upon install to point to the previous location: /usr/local/Cellar/qt@5.7/5.7.1  In this way, when qt@6.0/6.0 comes out, the original /usr/local/opt/qt will still be correct.  It's hopefully more or less future compatible.

Comment (vosst) [reply Formula/airmapd.rb]: Ah, that's nice.

**Comment by gpx1000** [comment]
Closing PR while I move things into smaller atomic checkins.

**Pull request closed without merging**